packages/orn-locator/src/resolvers/books.ts

```diff
@@ -107,6 +110,14 @@ const commonBook = memoize(async(id: string, version?: string, archive?: string)
   const default_page_slug = oswebData.webview_rex_link.match(/\/books\/.*\/pages\/(.*)$/)?.[1] as string;
   const default_page = default_page_slug && findTreeNodeBySlug(default_page_slug, archiveData.tree);
 
+  const subjects = oswebData.book_subjects ? oswebData.book_subjects.map(
+    ({ id, subject_name }: BookSubject) => ({ id, name: subject_name })
+  ) : [];
+
+  const categories = oswebData.book_categories ? oswebData.book_categories.map(
+    ({ id, subject_name, subject_category }: BookCategory) => ({ id, name: subject_category, subject_name })
```

Comment:
does this also include subject_id ? since you have the ids in the subject array that might be nice to include

Reply:
Doesn't look like it, this is an example response:

```
{
    id: 40,
    meta: { type: 'books.BookCategories' },
    subject_name: 'Science',
    subject_category: 'Biology'
  }
```